core/jobs/batch_jobs/math_interactions_audit_jobs.py

```diff
@@ -89,53 +191,65 @@ def run(self) -> beam.PCollection[job_run_result.JobRunResult]:
             | 'Final output' >> beam.Map(job_run_result.JobRunResult.as_stdout)
         )
 
-    def contains_math_interactions(
-            self, model: exp_models.ExplorationModel) -> bool:
-        """Checks if the exploration contains any state with any of the
-        math interactions.
 
-        Args:
-            model: ExplorationModel. Model instance to be checked.
+class FindMathExplorationsWithNonTrivialFunctionsJob(base_jobs.JobBase):
+    """Finds explorations that use at least one of the math interactions
```

Comment:
If you duplicate some of the stuff from your previous job it is also possible to create a `DoFn` to handle this (lookup `SendBatchMailchimpRequest` in the _mailchimp_population_jobs.py_

Reply:
I've refactored this into a util function. It seemed simpler.